GeeksCoreLibrary/Modules/Databases/Helpers/WiserTableDefinitions.cs

```diff
@@ -770,6 +770,26 @@ public class WiserTableDefinitions
                     new(WiserTableNames.WiserQuery, "idx_show_in_export_module", IndexTypes.Normal, new List<string> { "show_in_export_module" })
                 }
             },
+            
+            // wiser_styled_output
+            new WiserTableDefinitionModel()
+            {
+                Name = WiserTableNames.WiserStyledOutput,
+                LastUpdate = new DateTime(2023, 7, 18),
+                Columns = new List<ColumnSettingsModel>
+                {
+                    new("id", MySqlDbType.Int32, notNull: true, isPrimaryKey: true, autoIncrement: true),
+                    new("format_begin", MySqlDbType.MediumText),
+                    new("format_item", MySqlDbType.MediumText),
+                    new("format_end", MySqlDbType.MediumText),
+                    new("query_id", MySqlDbType.Int32),
+                    new("return_type", MySqlDbType.VarChar, 10)
+                },
+                Indexes = new List<IndexSettingsModel>
+                {
+                    new(WiserTableNames.WiserQuery, "idx_show_in_export_module", IndexTypes.Normal, new List<string> { "show_in_export_module" })
```

Comment:
You forgot to add the correct index here. This index is copied from `wiser_query`.

Reply:
![image](https://github.com/happy-geeks/geeks-core-library/assets/130973326/1da8944c-0f74-4ffb-990f-44483e086709)